stats.md

```diff
@@ -8,16 +8,8 @@ top_graphic: 3
 These statistics are updated periodically.
 
 <div class="figure">
-  <a href="https://plot.ly/~letsencrypt/7/" target="_blank" title="Active Certificates" style="display: block; text-align: center;"><img src="https://plot.ly/~letsencrypt/7.png" alt="Active Certificates" style="max-width: 100%;width: 600px;"  width="600" onerror="this.onerror=null;this.src='https://plot.ly/404.png';" /></a>
-  <script data-plotly="letsencrypt:7"  src="https://plot.ly/embed.js" async></script>
+  <div id="timeline" title="Active Certificates" style="display: block; text-align: center; max-width: 100%; width: 100%;">
 </div>
 
-<div class="figure">
-  <a href="https://plot.ly/~letsencrypt/9/" target="_blank" title="Certificates Issued and Revoked per day" style="display: block; text-align: center;"><img src="https://plot.ly/~letsencrypt/9.png" alt="Certificates Issued and Revoked per day" style="max-width: 100%;width: 600px;"  width="600" onerror="this.onerror=null;this.src='https://plot.ly/404.png';" /></a>
-  <script data-plotly="letsencrypt:9"  src="https://plot.ly/embed.js" async></script>
-</div>
-
-<div class="figure">
-  <a href="https://plot.ly/~letsencrypt/20/" target="_blank" title="Certificates by ~TLD" style="display: block; text-align: center;"><img src="https://plot.ly/~letsencrypt/20.png" alt="Certificates by ~TLD" style="max-width: 100%;width: 600px;"  width="600" onerror="this.onerror=null;this.src='https://plot.ly/404.png';" /></a>
-  <script data-plotly="letsencrypt:20"  src="https://plot.ly/embed.js" async></script>
-</div>
+<script src="/js/plotly-min.js"></script>
```

Comment:
I think both of these scripts can by async, if you make the fix I suggested in the non-blocking nit above.